backend/endpoints/rooms.py

```diff
@@ -50,48 +33,60 @@ async def create_room(
         member_id: A unique identifier of the member creating the room
 
     Returns:
-        A dict mapping the key room_id to the unique identifier (id) value of the room
-            {"room_id: "_id"}
-
-    Raises:
-        HTTP_424_FAILED_DEPENDENCY: "room creation unsuccessful" or
-            "unable to read database"
-        HTTP_406_NOT_ACCEPTABLE: raised when the maximium number of allowed users is exceeded
+        HTTP_200_OK (room already exist): {room_id}
+        HTTP_201_CREATED (new room created): {room_id}
+    Raises
+        HTTP_424_FAILED_DEPENDENCY: room creation unsuccessful
     """
     room_data = request.dict()
-    room_members = set(room_data["room_members"])
-    if len(room_members) > 9:
+    plugin_name = room_data.get("plugin_name")
+    room_name = room_data.get("room_name")
+    room_member_ids = list(room_data.get("room_members").keys())
+    rooms = await get_org_rooms(org_id=org_id, plugin=plugin_name)
+
+    if rooms is not None:
+        if plugin_name == Plugin.CHANNEL.value and room_name.casefold() in [
```

Comment:
Here we could do the same thing for `Plugin.CHANNEL` as suggested

Reply:
Done